modules/profile-store/pages/dotnet.adoc

```diff
@@ -291,6 +291,11 @@ For `FindActiveUsersByFirstName` it's a little trickier, because the N1QL is gen
 include::../examples/dotnet/UserProfileExample/UserProfileExample/Models/indexes.n1ql[tag=ix_findActiveUsersByFirstName]
 ----
 
+Creating the right indexes can be difficult. If you are having trouble creating the right index, here are two paths you can take to get help:
+
+* Starting in Couchbase Server 6.5, there is an index advisor that can automatically make index recommendations for you. To learn more about this, read the link:link:http://docs.couchbase.com/tutorials/profile-store/indexing.html[User Profile Indexing tutorial]. This tutorial uses the same type of data that you've already been creating if you've been following along with this User Profile Tutorial.
```

Comment:
```suggestion
* Starting in Couchbase Server 6.5, there is an index advisor that can automatically make index recommendations for you. To learn more about this, read the xref:indexing.adoc[User Profile Indexing tutorial]. This tutorial uses the same type of data that you've already been creating if you've been following along with this User Profile Tutorial.
```

It's generally preferable to use an xref for links to a page in the same module. See https://docs.couchbase.com/home/contribute/cross-references.html